src/xenia/base/cvar.cc

```diff
@@ -60,7 +79,11 @@ void ParseLaunchArguments(int& argc, char**& argv,
 
     auto result = options.parse(argc, argv);
     if (result.count("help")) {
-      PrintHelpAndExit();
+      if (xe::has_console_attached()) {
+        PrintHelpAndExit();
+      } else {
+        ExitWithNotification(true);
```

Comment:
Instead of the ExitWithNotification function, make 1 or 2 functions in logging.cc (or somewhere else if there's a better place): ShowInfoMessageBox(string) and ShowErrorMessageBox(string), and use those directly

Reply:
I did this in the latest push. Sorry for misunderstanding the previous comment on this.